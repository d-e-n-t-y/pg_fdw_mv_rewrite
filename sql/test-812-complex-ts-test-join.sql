set mv_rewrite.log_match_progress = 'true';
explain (verbose, costs off)
select * from events_all_room;
