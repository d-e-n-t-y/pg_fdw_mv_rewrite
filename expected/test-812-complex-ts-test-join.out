set mv_rewrite.log_match_progress = 'true';
explain (verbose, costs off)
select * from events_all_room;
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.test99
INFO:  mv_rewrite_check_group_clauses_for_mv: looking to rewrite JOIN, but MV is GROUP BY.
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.events_all_room_mv
INFO:  mv_rewrite_join_node_is_valid_for_plan_recurse: no match found in plan for MV rtindex: 3; MV Oid: 0
INFO:  mv_rewrite_join_node_is_valid_for_plan_recurse: left side of join is not valid for plan: {RANGETBLREF :rtindex 3}
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.test99
INFO:  mv_rewrite_check_group_clauses_for_mv: looking to rewrite JOIN, but MV is GROUP BY.
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.events_all_room_mv
INFO:  mv_rewrite_join_node_is_valid_for_plan_recurse: no match found in plan for MV rtindex: 3; MV Oid: 0
INFO:  mv_rewrite_join_node_is_valid_for_plan_recurse: left side of join is not valid for plan: {RANGETBLREF :rtindex 3}
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.test99
INFO:  mv_rewrite_check_group_clauses_for_mv: looking to rewrite JOIN, but MV is GROUP BY.
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.events_all_room_mv
INFO:  mv_rewrite_add_rewritten_mv_paths: creating and adding path for scan on: public.events_all_room_mv
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.test99
INFO:  mv_rewrite_check_group_clauses_for_mv: looking to rewrite JOIN, but MV is GROUP BY.
INFO:  mv_rewrite_add_rewritten_mv_paths: evaluating MV: public.events_all_room_mv
INFO:  mv_rewrite_add_rewritten_mv_paths: creating and adding path for scan on: public.events_all_room_mv
                                                QUERY PLAN                                                 
-----------------------------------------------------------------------------------------------------------
 Custom Scan (MVRewriteScan)
   Output: e."timestamp", e.hc1, e.hc2, e.category, e.subcategory, e.advices, e.warnings, e.detail, r.name
   Rewritten: scan of public.events_all_room_mv
(3 rows)

